README.md

```diff
@@ -1,3 +1,4 @@
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/13dd86379fff49d3b96fd3f3d954d4e8)](https://www.codacy.com/app/dhvatta/triplea?utm_source=github.com&utm_medium=referral&utm_content=triplea-game/triplea&utm_campaign=badger)
```

Comment:
I think you mentioned this in another issue or PR, but there's a plan to move from your personal account to an org-managed account, right?